From 05c0b3fef0849cdb90de63bcc4065d87dbb3f9bb Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Tue, 6 Sep 2016 13:19:37 +0800
Subject: [PATCH 1152/5242] MLK-13188-2 dts: imx6ull: change the usdhc root
 clock to 396MHz

commit  21155723b2427238ae71b2395111dab67d9d9e60 from
https://source.codeaurora.org/external/imx/linux-imx.git

Due to the errata ERR010450 limit, this patch change the imx6ull
usdhc root clock to 132MHz in soc related dts file, remove all
the root clock setting in board dts file, after this patch,
SDR104/HS200 work at 132MHz, DDR50/DDR52 work at 33MHz.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-emmc.dts |    3 ---
 arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts       |    4 ----
 2 files changed, 7 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-emmc.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-emmc.dts
index 69323b6..934e6f6 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-emmc.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-emmc.dts
@@ -12,9 +12,6 @@
 	pinctrl-0 = <&pinctrl_usdhc1>;
 	pinctrl-1 = <&pinctrl_usdhc1_100mhz>;
 	pinctrl-2 = <&pinctrl_usdhc1_200mhz>;
-	assigned-clocks = <&clks IMX6UL_CLK_USDHC1_SEL>, <&clks IMX6UL_CLK_USDHC1>;
-	assigned-clock-parents = <&clks IMX6UL_CLK_PLL2_PFD0>;
-	assigned-clock-rates = <0>, <176000000>;
 	cd-gpios = <>;
 	wp-gpios = <>;
 	vmmc-supply = <>;
diff --git a/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts b/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts
index bf8db20..4ea3d91 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts
@@ -13,10 +13,6 @@
 	pinctrl-0 = <&pinctrl_usdhc2_8bit>;
 	pinctrl-1 = <&pinctrl_usdhc2_8bit_100mhz>;
 	pinctrl-2 = <&pinctrl_usdhc2_8bit_200mhz>;
-	assigned-clocks = <&clks IMX6UL_CLK_USDHC2_SEL>, <&clks IMX6UL_CLK_USDHC2>;
-	assigned-clock-parents = <&clks IMX6UL_CLK_PLL2_PFD2>;
-	assigned-clock-rates = <0>, <396000000>;
-	max-frequency = <132000000>;
 	bus-width = <8>;
 	non-removable;
 	status = "okay";
-- 
1.7.9.5

